docs/how-to/publish/authenticate.rst

```diff
@@ -0,0 +1,96 @@
+.. _how-to-authenticate:
+
+Authenticate
+============
+
+To manage and publish snaps on a snap store, you must be logged in to a Snapcraft
+developer account. Snapcraft account credentials can also be exported for use on systems
+where login is not possible or desired, such as offline systems.
+
+
+Log in to your Snapcraft account
+--------------------------------
+
+To log in to your Snapcraft developer account from the terminal, run:
+
+.. code-block:: bash
+
+    snapcraft login
+
+This will prompt you to enter your email, password, and, if applicable, your second
+factor authentication code.
+
+
+Export Snapcraft account credentials
+------------------------------------
+
+To export Snapcraft account credentials, run the ``export-login`` command followed
+by a name for the resulting credentials file:
+
+.. code-block:: bash
+
+    snapcraft export-login <credentials-filename>
+
+Like ``snapcraft login``, this will prompt you to enter your email, password, and second
+factor authentication code. Once entered, your account credentials will be exported to a
+file in your current directory with the name you specified.
+
+Note that credentials exported with Snapcraft 7 can only be used with Snapcraft 7 or
+greater.
```

Comment:
not sure if this bit could be served by one of those quote blocks

Reply:
I agree in this particular case. And it applies to everything in this section, so the top of the section is probably the proper place for it.